unicodetools/src/main/java/org/unicode/text/UCD/ToolUnicodePropertySource.java

```diff
@@ -1177,35 +1177,25 @@ public int getMaxWidth(boolean isShort) {
                             .addAll(script.getSet("Orya"))
                             .addAll(script.getSet("Beng"))
                             .addAll(script.getSet("Deva"));
+            final UnicodeSet incbLinker =
+                    conjunctLinkingScripts
+                            .cloneAsThawed()
+                            .retainAll(isc.getSet(Indic_Syllabic_Category_Values.Virama));
+            final UnicodeSet incbConsonant =
+                    conjunctLinkingScripts
+                            .cloneAsThawed()
+                            .retainAll(isc.getSet(Indic_Syllabic_Category_Values.Consonant));
             final UnicodeMap<String> incbDefinition =
                     new UnicodeMap<String>()
                             .setErrorOnReset(true)
-                            .putAll(
-                                    conjunctLinkingScripts
-                                            .cloneAsThawed()
-                                            .retainAll(
-                                                    isc.getSet(
-                                                            Indic_Syllabic_Category_Values.Virama)),
-                                    "Linker")
-                            .putAll(
-                                    conjunctLinkingScripts
-                                            .cloneAsThawed()
-                                            .retainAll(
-                                                    isc.getSet(
-                                                            Indic_Syllabic_Category_Values
-                                                                    .Consonant)),
-                                    "Consonant")
+                            .putAll(incbLinker, "Linker")
+                            .putAll(incbConsonant, "Consonant")
                             .putAll(
                                     gcb.getSet("Extend")
                                             .removeAll(ccc.getSet("Not_Reordered"))
```

Comment:
@eggrobin This still removes ccc=0 so
- [[179-C31](https://www.unicode.org/cgi-bin/GetL2Ref.pl?179-C31)] Consensus: Change the derivation of InCB=Extend to [\p{gcb=Extend}\p{gcb=ZWJ}-\p{InCB=Linker}-\p{InCB=Consonant}-[\u200c]]. For Unicode Version 16.0. See document [L2/24-064](https://www.unicode.org/cgi-bin/GetMatchingDocs.pl?L2/24-064) item 5.8.

is not done yet in this code, right?